src/postman/transformers/request.ts

```diff
@@ -23,7 +23,7 @@ export function transformQueryParam(queryParam: QueryParam): IHttpQueryParam {
 
 export function transformHeader(header: Header): IHttpHeaderParam {
   return {
-    name: header.key,
+    name: header.key.toLowerCase(),
```

Comment:
I am almost sure that the `Header` object in Postman collection is already returning it in lower case, so you do not have to take care of it. Can you verify and if so, remove this part?

Reply:
I verified once again and Postman Collection conversion does not touch the case of letters here.

Reply:
Damn postman library

Reply:
Damn indeed